src/parsec_client/operations/list_providers.md

```diff
@@ -1,10 +1,36 @@
 # ListProviders
 
-## Opcode: 8 (decimal), 0x0008 (hex)
+Gets a list of available Parsec providers to be used by clients. Opcode: 8 (`0x0008`)
 
-## Summary
+## Parameters
 
-Gets a list of available Parsec providers to be used by clients.
+No parameters are needed for this operation.
+
+## Results
+
+| Name        | Type                                           | Description                  |
+|-------------|------------------------------------------------|------------------------------|
+| `providers` | Vector of [`ProviderInfo`](#providerinfo-type) | List of provider information |
+
+### ProviderInfo type
+
+A `ProviderInfo` type contains the following members:
+
+| Name          | Type             | Description                                                               |
+|---------------|------------------|---------------------------------------------------------------------------|
+| `uuid`        | String           | Unique, permanent, identifier of the provider                             |
```

Comment:
Mind adding that this is a [version 4 UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random))?